product/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+from core      import TimeStampModel
+
+# Create your models here.
+class Main_Category(models.Model):
+    name=models.CharField(max_length=50)
+    
+    class Meta:
+     db_table='main_categories'
+     
+    def __str__(self):
+            return self.main_categories_name
```

Comment:
- class 이름에는 _ 를 사용하지 않습니다.
- indent가 확인해 주세요. 

```suggestion
class MainCategory(models.Model):
    name = models.CharField(max_length=50)
    
    class Meta:
        db_table='main_categories'
     
    def __str__(self):
        return self.main_categories_name
```